src/main/java/rx/Observable.java

```diff
@@ -5577,14 +5577,16 @@ public final void forEach(final Action1<? super T> onNext, final Action1<Throwab
     
     /**
      * Groups the items emitted by an {@code Observable} according to a specified criterion, and emits these
-     * grouped items as {@link GroupedObservable}s, one {@code GroupedObservable} per group.
+     * grouped items as {@link GroupedObservable}s. If a {@code GroupedObservable} 
+     * terminates before the source terminates then the next emission by the source with the same key as the 
```

Comment:
GroupedObservable allows only a single Subscriber during its lifetime and if this Subscriber unsubscribes before the main source terminates, the next emission by the main source having the same key will trigger a new GroupedObservable emission.